<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TcpSocketView</class>
 <widget class="QWidget" name="TcpSocketView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1124</width>
    <height>786</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" columnstretch="4,1">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>消息记录</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" rowstretch="1">
      <item row="0" column="0" colspan="2">
       <widget class="QTextBrowser" name="textBrowser"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>服务器信息</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0">
       <widget class="QCheckBox" name="pauseDisplay_checkBox">
        <property name="text">
         <string>暂停显示</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>服务器端口：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="hexRecive_checkBox">
        <property name="text">
         <string>16进制接收</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QPushButton" name="saveRecive_pushButton">
        <property name="text">
         <string>保存数据</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="hexSend_checkBox">
        <property name="text">
         <string>16进制发送</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLineEdit" name="serverIp_lineEdit">
        <property name="text">
         <string>127.0.0.1</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QSpinBox" name="serverPort_spinBox">
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>8888</number>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QPushButton" name="connectServer_pushButton">
        <property name="text">
         <string>连接</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>服务器地址：</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="clearRecive_pushButton">
        <property name="text">
         <string>清空接收</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>发送消息</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4" columnstretch="4,1">
      <item row="1" column="1">
       <widget class="QSpinBox" name="timeInterval_spinBox">
        <property name="specialValueText">
         <string/>
        </property>
        <property name="suffix">
         <string> ms</string>
        </property>
        <property name="prefix">
         <string/>
        </property>
        <property name="maximum">
         <number>10000000</number>
        </property>
        <property name="singleStep">
         <number>100</number>
        </property>
        <property name="stepType">
         <enum>QAbstractSpinBox::DefaultStepType</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="regularlySend_checkBox">
        <property name="text">
         <string>定时发送</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="send_pushButton">
        <property name="text">
         <string>发送</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+Return</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="3">
       <widget class="QPlainTextEdit" name="plainTextEdit"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>clearRecive_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>textBrowser</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>994</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>738</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
